google-cloud-errorreporting/src/test/java/com/google/cloud/errorreporting/spi/v1beta1/MockErrorGroupService.java

```diff
@@ -34,6 +34,16 @@ public MockErrorGroupService() {
   }
 
   @Override
+  public void addResponse(GeneratedMessageV3 response) {
+    serviceImpl.addResponse(response);
+  }
+
+  @Override
+  public void addException(Exception exception) {
+    serviceImpl.addException(exception);
+  }
+
+  @Override
```

Comment:
Could you remove the `@Override` on `setResponses` so that we can safely easily it from the interface?